### DISCUSSION TITLE
Find entries only in older kdbx file versions

### DISCUSSION THREAD

**Discussion by Redsandro** [keepassxreboot/keepassxc#7560] [Q&A] [answered]
We have been sharing a kdbx file over Dropbox and now we have a lot of conflicted copies. 

This means someone added an entry, and then someone else added an entry before the previous one was properly synced, so one of them becomes renamed in the next sync and the latest file will only contain one of the two entries. It went on for a while and a couple of entries are now missing from the latest version of the file.

Is there a way to automate finding out what entries exist in older versions of a kdbx file but not in the newer? Is there a way to automate this with a dozen or so kdbx files?

**Comment by droidmonkey** [comment, marked as answer]
Just merge all the conflicts into one database. Do a search for `*` and sort by modified date. You can look at the entry history of the recently modifies entries to see what changed.
